### ISSUE TITLE
In OS X Contacts.app (9.0), server contacts don't sync to Mac until you create one on Mac

### ISSUE THREAD

**Issue by smoyte** [inferiorhumanorgans/meishi#17]
When you first add the CardDAV account it hits the server and does PROPFINDs but never GETs. Then when you add a contact to Contacts.app, after it does the PUT, for some reason it notices the server contacts and pulls them down.

My use case is read only from the server so this obviously is problematic. Any thoughts?

Thanks for the great app!

**Comment by inferiorhumanorgans** [comment]
It could be an issue with trailing slashes and Contacts.app not recognizing a collection.  What URL does Contacts.app run a PROPFIND on initially?

**Comment by smoyte** [comment]
Hi there! Thanks for the reply. Unfortunately I can't remember and I've since switched to using this: https://github.com/schmurfy/dav4rack_ext

I'm still working out the kinks but it fit my use case a bit better (trying to add something to an existing app). Hoping to write up my setup once it's done as I feel like others might want to expose an existing list of users in a Rails app over CardDAV. Couldn't find anything out there for that...

**Comment by inferiorhumanorgans** [comment]
Yeah this was mostly meant as a standalone proof-of-concept, not really something to integrate as I library.  Unfortunately one of the hardest parts is that "older" apps (especially Apple ones) are just very buggy.

I'm giving some thought to rewriting this with Rocket.rs or Phoenix though.
